SMLHelper/Handlers/CraftDataHandler_BelowZero.cs

```diff
@@ -347,16 +286,7 @@ RecipeData ICraftDataHandler.GetModdedRecipeData(TechType techType)
         /// <param name="equipmentType">The EquipmentType for that TechType.</param>
         void ICraftDataHandler.SetEquipmentType(TechType techType, EquipmentType equipmentType)
         {
-            if (CraftDataPatcher.CustomTechData.ContainsKey(techType))
-                CraftDataPatcher.CustomTechData[techType][TechData.PropertyToID("equipmentType")] = new JsonValue((int)equipmentType);
-            else
-                CraftDataPatcher.CustomTechData[techType] = new JsonValue
-                {
-                    {
-                        TechData.PropertyToID("equipmentType") ,
-                        new JsonValue((int)equipmentType)
-                    }
-                };
+            AddJsonProperty(techType, "equipmentType", new JsonValue((int)equipmentType));
```

Comment:
I think, for the sake of perfection, you can add intermediate generic method and put `new JsonValue()` there. Other than that, it looks good.

Reply:
That was my first instinct.  
Turns out, it isn't feasible since the `JsonValue` constructor only accepts a specific set of types, which don't really have anything in common to make for a clean constraint.

This would have ended up with a separate overload of the same method for each type that `JsonValue` supports.  
Rather than do that and end up with almost as much nearly duplicate code as before, I opted to just require the entire `JsonValue` instance and leave that to be handled one level up.

Reply:
Ah, I see.. Is JsonValue is specific to BelowZero? I tried to find it before commenting, but it seems it's not part of Newtonsoft.Json and I don't have BZ yet.